### DISCUSSION TITLE
Is there a roadmap? If not, how can I avoid being annoying?

### DISCUSSION THREAD

**Discussion by danxcraig** [huntabyte/shadcn-svelte#1031] [Help] [answered]
While searching for a navigation-menu in shadcn-svelte docs, I ended up down the github rabbit hole and ended up [here](https://github.com/huntabyte/shadcn-svelte/issues/881).

I understand the Guinness answer @huntabyte and @shyakadavis have given on various issue threads (i.e. don't worry about it, good things come to those who wait!), but those answers aren't practically helpful to a problem I often have (and I'm sure others do too):

_When something is not available and I need it soon, should I build it, contribute to it, request it, pay for it, or wait for it?_

The problem with the Guinness answer is that it doesn't give me the information I need to make a good decision. Does it mean "at some point, but idk when", "this year", "it's coming tomorrow", "listen dude go jump, I'll get to it when I get to it"...or some/all of the above dependent on the case?

In the absence of clear information, I feel like my only option is to either waste my time building something that is already being built, to pay for something I don't need to, or to waste my time leaving messages like this one, all of which aren't good for the community and aren't good for me and wastes your time too.

So...short story long; is there a roadmap (because I can't find one), and if not, what's the most appropriate way for a non-contributor to figure out the status of something they need, short of sending annoying messages and creating annoying new issues all the time?

**Comment by huntabyte** [comment, marked as answer]
There isn't an official roadmap with dates/times.  I will say that the Navigation Menu likely won't land until the Svelte 5 version of Bits UI/shadcn-svelte (which is just around the corner). If you need this before then, we welcome contributions to Melt UI (https://github.com/melt-ui/melt-ui), or of course, feel free to build it yourself for your own use.

**Reply by npetrangelo** [reply]
I was wondering, are Svelte 5 versions in the works using the release candidate for Svelte 5? Are they on a branch somewhere? Svelte Society recently [released a video](https://youtu.be/xCeYmdukOKI?si=K1Hl3zkGpfanpIpz) suggesting that libraries (such as this) begin porting things over sooner rather than later.
